mkt/ratings/views.py

```diff
@@ -67,17 +68,21 @@ def filter_queryset(self, queryset):
         Custom filter method allowing us to filter on app slug/pk and user pk
         (or the special user value "mine"). A full FilterSet is overkill here.
         """
-        filters = {}
+        filters = Q()
         app = self.request.GET.get('app')
         user = self.request.GET.get('user')
+        lang = self.request.GET.get('lang')
+        match_lang = self.request.GET.get('match_lang')
         if app:
             self.app = self.get_app(app)
-            filters['addon'] = self.app
+            filters &= Q(addon=self.app)
         if user:
-            filters['user'] = self.get_user(user)
+            filters &= Q(user=self.get_user(user))
+        elif lang and match_lang == '1':
+            filters &= Q(user__lang=lang)
```

Comment:
Yeah, I must have removed it. I was originally using both the `1` and `0` case but now I'm just using the `1` case. I'll update the docs and remove that test since it isn't being used.